src/aiy/_drivers/_alsa.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helpers for ALSA tools, including aplay and arecord."""
+
+def sample_width_to_string(sample_width):
+    """Convert sample width (bytes) to ALSA format string."""
+    return {1: 's8', 2: 's16', 4: 's32'}[sample_width]
```

Comment:
this will raise a KeyError if you use a bad sample_width. An alternative would be to write:
{1: 's8', 2: 's16', 4: 's32'}.get(sample_width, None) or handle the key-error and re-throw as ValueError.